src/methods/SANA.cpp

```diff
@@ -2865,14 +2868,24 @@ void SANA::initIterPerSecond() {
     initializedIterPerSecond = true;
 
     cout << "Determining iteration speed...." << endl;
-    long long int iter = 1E6;
-    hillClimbingIterations(iter - 1);
-    double res = iter/timer.elapsed();
-    cout << "SANA does " << to_string(res)
-         << " iterations per second (took " << timer.elapsedString()
-         << " doing " << iter << " iterations)" << endl;
-
-    iterPerSecond = res;
+    //long long int iter = 1E6;
```

Comment:
Don't just comment out the code. Delete it if you're confident your replacement works. It'll always be in the git history.